build/package/scripts/build-java.sh

```diff
@@ -60,53 +55,33 @@ echo " that this build expects generated application artifacts to be copied to."
 echo " The project gradle script should read this env var to copy all the "
 echo " generated application artifacts."
 echo
-echo " NEXUS_* env vars:"
-echo " following env vars NEXUS_HOST, NEXUS_USER and NEXUS_PASSWORD"
-echo " are available and should be read by your gradle script."
-echo
 export ODS_OUTPUT_DIR=${OUTPUT_DIR}
 echo "Exported env var 'ODS_OUTPUT_DIR' with value '${OUTPUT_DIR}'"
 echo
 echo "Building (Compile and Test) ..."
 ./gradlew clean build ${GRADLE_ADDITIONAL_TASKS} ${GRADLE_OPTIONS}
 echo
 
-if [ "${DEBUG}" == "true" ]; then
-  set -x
-  echo
-  echo "List content of ${OUTPUT_DIR}"
-  ls -lart "${OUTPUT_DIR}"
-  echo
-fi
-
 echo "Verifying unit test report was generated  ..."
 BUILD_DIR="build"
 UNIT_TEST_RESULT_DIR="${BUILD_DIR}/test-results/test"
 
-if [ "$(ls -A ${UNIT_TEST_RESULT_DIR})" ]; then
+if [ "$(ls -d ${UNIT_TEST_RESULT_DIR})" ]; then
```

Comment:
`if [ -d "${UNIT_TEST_RESULT_DIR}" ]; then`

Reply:
got it :-)